### PR TITLE
Dev 6 pr corrections on master

### PR DESCRIPTION
**Pull request by chaimchessrak-vayyar**

### CODE REVIEW

File: scripts/skeleton.py [hunk 1]
```diff
@@ -17,9 +28,12 @@ def parse_text_file(path_to_read_from):
 
 def separate_un_relevant_lines(chips_map_as_str):
     """
+    Get content of text file and separate it into the wafer
+    part and the rest of the text.
```
Comment (alons-vayyar) [inline scripts/skeleton.py:32]: add space row.

File: scripts/skeleton.py [hunk 2]
```diff
@@ -32,20 +46,23 @@ def separate_un_relevant_lines(chips_map_as_str):
     chips_map_part_string = '\n'.join(chips_map_part_list)
     rest_of_the_text = chips_map_as_str.replace(chips_map_part_string, '$wafer')
     rest_of_text_as_template = Template(rest_of_the_text)
+    logger.debug('End of separate wafer from text.')
```
Comment (alons-vayyar) [inline scripts/skeleton.py:49]: As we talked,
change the wording of the logs.(all of them)

File: scripts/skeleton.py (hunk 1 above)
Comment (chaimchessrak-vayyar) [reply scripts/skeleton.py:32]: Where?

File: scripts/skeleton.py (hunk 2 above)
Comment (chaimchessrak-vayyar) [reply scripts/skeleton.py:49]: I already changed this message now I understand that you want 
some more changes but I didn't know what bad with the current 
words so I don't know what to change.

**Pull request merged**